media-source-respec.html

```diff
@@ -154,288 +154,288 @@ <h3>
-              access points for multiplexed streams.
-            </p>
-          </dd>
-          <dt>
-            <dfn>SourceBuffer byte stream format specification</dfn>
-          </dt>
-          <dd>
-            <p>
-              The specific [=byte stream format specification=] that describes the format of the
-              byte stream accepted by a {{SourceBuffer}} instance. The [=byte stream format
-              specification=], for a {{SourceBuffer}} object, is initially selected based on the
-              |type:DOMString| passed to the {{MediaSource/addSourceBuffer()}} call that created
-              the object, and can be updated by {{SourceBuffer/changeType()}} calls on the object.
-            </p>
-          </dd>
-          <dt>
-            <dfn>`SourceBuffer` configuration</dfn>
-          </dt>
-          <dd>
-            <p>
-              A specific set of tracks distributed across one or more {{SourceBuffer}} objects
-              owned by a single {{MediaSource}} instance.
-            </p>
-            <p>
-              Implementations MUST support at least 1 {{MediaSource}} object with the following
-              configurations:
-            </p>
-            <ul>
-              <li>A single SourceBuffer with 1 audio track and/or 1 video track.
-              </li>
-              <li>Two SourceBuffers with one handling a single audio track and the other handling a
-              single video track.
-              </li>
-            </ul>
-            <p>
-              MediaSource objects MUST support each of the configurations above, but they are only
-              required to support one configuration at a time. Supporting multiple configurations
-              at once or additional configurations is a quality of implementation issue.
-            </p>
-          </dd>
-          <dt>
-            <dfn>Track Description</dfn>
-          </dt>
-          <dd>
-            <p>
-              A byte stream format specific structure that provides the [=Track ID=], codec
-              configuration, and other metadata for a single track. Each track description inside a
-              single [=initialization segment=] has a unique [=Track ID=]. The user agent MUST run
-              the [=append error=] algorithm if the [=Track ID=] is not unique within the
-              [=initialization segment=].
-            </p>
-          </dd>
-          <dt>
-            <dfn>Track ID</dfn>
-          </dt>
-          <dd>
-            <p>
-              A Track ID is a byte stream format specific identifier that marks sections of the
-              byte stream as being part of a specific track. The Track ID in a [=track
-              description=] identifies which sections of a [=media segment=] belong to that track.
-            </p>
-          </dd>
-        </dl>
-      </section>
+    </section>
+    <section id="definitions">
```

Comment:
Seems a good opportunity to drop the ID. ReSpec generates the same ID, so that does not change anything.

```suggestion
    <section>
```

Reply:
Thanks, done.